### ISSUE TITLE
Force refreshing?

### ISSUE THREAD

**Issue by IBPX** [tootsuite/mastodon#219]
Is there any way to force Mastodon to check for a user's new posts? Or a global refresh? Some of the accounts I'm following on a test account are followed by no one on Mastodon but me, and it _shows_ the last post as being from a few months ago, even though the account has posted many times since.

Upon further investigation, I do see new posts from them, but there is a huge gap. Is there a way to see those posts that didn't get sent to Mastodon? (by Salmon?)

**Comment by Gargron** [comment]
In a way this is a duplicate of #34, pulling content instead of having it pushed to us. It is blocked by the same limitations too.
